docs/api/target_properties.rst

```diff
@@ -169,6 +181,14 @@ with the corresponding :ref:`blt_add_executable`, :ref:`blt_add_library`,
 Output is of the form for each property:
  | [<target> property] <property>: <value>
 
+.. code-block:: cmake
+    :caption: **Example**
+    :linenos:
+
+    blt_print_target_properties(TARGET foo)
+    blt_print_target_properties(TARGET foo CHILDREN TRUE)
+    blt_print_target_properties(TARGET foo CHILDREN TRUE PROPERTY_NAME_REGEX "CXX")
+    blt_print_target_properties(TARGET foo PROPERTY_NAME_REGEX "CXX" PROPERTY_VALUE_REGEX "ON")
```

Comment:
@white238  Can you activate this branch on ReadTheDocs to verify the formatting is alright?